ui/src/keyboardMappings.ts

```diff
@@ -41,9 +48,21 @@ export const keys = {
   F11: 0x44,
   F12: 0x45,
   F13: 0x68,
+  F14: 0x69,
+  F15: 0x6a,
+  F16: 0x6b,
+  F17: 0x6c,
+  F18: 0x6d,
+  F19: 0x6e,
+  F20: 0x6f,
+  F21: 0x70,
+  F22: 0x71,
+  F23: 0x72,
+  F24: 0x73,
   Home: 0x4a,
+  HashTilde: 0x32,  // non-US # and ~
```

Comment:
Filling in the missing KeyStroke codes based on [Section 10. Keyboard/Keypad Page 0x07 on page 82-89](https://usb.org/sites/default/files/hut1_21.pdf)

There's still more to go when we need better VirtualKeyboard layout support, but this is a good time to make sure there are no holes.